sdk/tests/conftest.py

```diff
@@ -38,48 +37,6 @@ def pytest_collection_modifyitems(items):
         item.add_marker(pytest.mark.vcr)
         item.fixturenames.append("patched_vcr_object_handling")
```

Comment:
can you try running the tests with this gone as well? if we are lucky this is no longer needed as well. It was due to a bug around multipart request handling in vcr.